compass/src/components/Applications/Applications.component.js

```diff
@@ -54,8 +55,10 @@ class Applications extends React.Component {
     >
       <b>{application.name}</b>
     </span>,
-    application.description,
-    application.labels ? this.createLabels(application.labels) : '-',
+    application.description ? application.description : EMPTY_TEXT_PLACEHOLDER,
+    application.labels
```

Comment:
```suggestion
    application.labels && Object.keys(application.labels).length 
```
A rather nitpicking change, if you feel it looks good as it is now, please leave it.